src/elements/hx-beacon/index.js

```diff
@@ -0,0 +1,14 @@
+import { HXElement } from '../../interfaces/HXElement/index.js';
```

Comment:
Please add `<hx-beacon>` component unit tests to `src/elements/hx-beacon/index.spec.js`.

Reply:
Changes `have been done, please review